packages/core/onKeyStroke/index.ts

```diff
@@ -9,6 +9,12 @@ export interface OnKeyStrokeOptions {
   eventName?: KeyStrokeEventName
   target?: MaybeComputedRef<EventTarget | null | undefined>
   passive?: boolean
+  /**
+   * Set to `false` to ignore repeated strokes.
+   *
+   * @default true
+   */
+  repeat?: boolean
```

Comment:
This is too general, to me, it's more like "we will fire the event" repeatedly.

Instead, I would call it `dedupe` and default to `false`.

Reply:
I agree with that, but I just found a conventional name `autoRepeat` that seems adopted by public:
1. https://developer.apple.com/documentation/uikit/uistepper/1624079-autorepeat
2. https://learn.microsoft.com/en-us/office/vba/api/access.commandbutton.autorepeat
3. https://doc.qt.io/qt-6/qabstractbutton.html#autoRepeat-prop